content/get-started/getting-started-with-git/caching-your-github-credentials-in-git.md

```diff
@@ -34,7 +34,7 @@ For more information about authenticating with {% data variables.product.prodnam
 
 ## Git Credential Manager
 
-[Git Credential Manager](https://github.com/GitCredentialManager/git-credential-manager) (GCM) is another way to store your credentials securely and connect to GitHub over HTTPS. With GCM, you don't have to manually [create and store a {% data variables.product.pat_generic %}](/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens), as GCM manages authentication on your behalf, including 2FA (two-factor authentication).
+[Git Credential Manager](https://github.com/GitCredentialManager/git-credential-manager) (GCM) extends your Git installation. On Git's behalf, it helps you log in to {% data variables.product.product_name %}, including 2FA (two-factor authentication), and stores your credentials securely. Or, if you have manually [created and stored a {% data variables.product.pat_generic %}](/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens), GCM can store that token securely, and provided it to {% data variables.product.product_name %} automatically when needed. In Git terms, GCM is an [OAuth credential helper](https://git-scm.com/docs/gitcredentials).
```

Comment:
Again, I think the language here is a little misleading:

- GCM is a `git` credential manager that can be used with GitHub or other `git` platforms
- Because of this, GCM isn't storing OAuth credentials, but storing credentials used with `git`
- The _"GCM can store that token securely"_ implies or infers that users could insecurely store credentials in GCM, which I don't know is true and might be misleading

Reply:
I think the thing that's tripping up around OAuth is that `gh` and `gcm` can both act as [oauth credential helpers](https://git-scm.com/docs/gitcredentials#_oauth). Andy's right that the credential helpers store _whatever_ credential is used to authenticate git. Optionally, some credential helpers can perform an OAuth flow for you to obtain a token so that you don't need to create one yourself.

Reply:
The topic of this part of the docs is **Caching your GitHub credentials in Git**, as part of **Getting started with Git**. I think the existing docs don't explain well enough the three points @andyfeller lists about how credentials, GitHub, and git interact. They instead tell users how to set up one mechanism works. 

Both explaining and instructing "how to" are valid goals for documentation, but they are not the same thing. See the [Diátaxis framework](https://diataxis.fr/) for what I think is a pretty good explanation of the difference.

The purpose of my PR was essentially to improve the explanation aspect of this page of the docs. I sense that the pushback relates to keeping this page a "how to". GitHub should make a choice about what the goal of this page is.